src/hotspot/share/gc/g1/g1RootProcessor.cpp

```diff
@@ -108,15 +103,10 @@ void G1RootProcessor::process_strong_roots(OopClosure* oops,
   process_java_roots(&closures, NULL, 0);
   process_vm_roots(&closures, NULL, 0);
 
-#ifdef ASSERT
-  // only for verification purpose
-  // already processed in java roots.
-  _process_strong_tasks.try_claim_task(G1RP_PS_CodeCache_oops_do);
-  // inside safe point
-  _process_strong_tasks.try_claim_task(G1RP_PS_refProcessor_oops_do);
-#endif
-
-  _process_strong_tasks.all_tasks_completed(n_workers());
+  // CodeCache is already processed in java roots
+  // refProcessor is not needed since we are inside a safe point
+  _process_strong_tasks.all_tasks_completed(n_workers(),
+      G1RP_PS_CodeCache_oops_do, G1RP_PS_refProcessor_oops_do);
```

Comment:
When parameters or arguments are on multiple lines, we usually align all the arguments, and usually one per line.  There are some other similar cases elsewhere in this change that I didn't redundantly comment.